src/app/client/src/app/modules/program-dashboard/components/program-datasets/program-datasets.component.ts

```diff
@@ -92,7 +92,8 @@ export class DatasetsComponent implements OnInit, OnDestroy {
   minEndDate: any;  //Min end date - has to be one more than start date 
   maxEndDate: any;  //Max end date -  current date has to be max
   maxStartDate: any; //Start date - has to be one day less than end date
-
+  displayFilters:any = {};
+  loadash = _;
```

Comment:
why are we creating a variable to store the entire loadash

Reply:
to iterate an object using loadash in the template (as previously has been done in the admin dashboard for selected filters) @rajeevsathish